public-ui/src/app/views/ehr-view/ehr-view.component.ts

```diff
@@ -184,11 +185,12 @@ export class EhrViewComponent implements OnInit, OnDestroy {
     this.resetSelectedGraphs();
     this.graphToShow = g;
     if (this.graphToShow === GraphType.Sources) {
+      this.treeLoading = true;
       this.subscriptions.push(this.api.getCriteriaRolledCounts(r.conceptId)
-        .subscribe({
+      .subscribe({
           next: result => {
             this.treeData = [result.parent];
-            this.treeData['children'] = result.children.items;
```

Comment:
where is this being set now ?

Reply:
It was redundant. It didn't need it to get the next level children.